specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.2/TextAnalytics.json

```diff
@@ -0,0 +1,3477 @@
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "InvalidRequest",
+            "InvalidArgument",
+            "InternalServerError",
+            "ServiceUnavailable",
+            "NotFound"
+          ],
+          "x-ms-enum": {
+            "name": "ErrorCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message."
+        },
+        "target": {
+          "type": "string",
+          "description": "Error target."
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError",
+          "description": "Inner error contains more specific information."
+        },
+        "details": {
+          "type": "array",
+          "description": "Details about specific errors that led to this reported error.",
+          "items": {
+            "$ref": "#/definitions/TextAnalyticsError"
+          }
+        }
+      }
+    },
+    "TextAnalyticsWarning": {
+      "type": "object",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "LongWordsInDocument",
+            "DocumentTruncated"
+          ],
+          "x-ms-enum": {
+            "name": "WarningCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Warning message."
+        },
+        "targetRef": {
+          "type": "string",
+          "description": "A JSON pointer reference indicating the target object."
+        }
+      }
+    },
+    "ExtractiveSummarizationTask": {
+      "type": "object",
+      "properties": {
+        "parameters": {
+          "type": "object",
+          "properties": {
+            "model-version": {
+              "default": "latest",
+              "type": "string"
+            },
+            "loggingOptOut": {
+              "default": false,
+              "type": "boolean"
+            },
+            "stringIndexType": {
+              "$ref": "#/definitions/StringIndexType"
+            },
+            "sentenceCount": {
+              "default": 3,
+              "type": "integer"
+            },
+            "sortBy": {
+              "default": "Offset",
+              "enum": [
+                "Offset",
+                "Rank"
+              ],
+              "type": "string"
+            }
+          }
+        },
+        "taskName": {
+          "type": "string"
+        }
+      }
+    },
+    "CustomEntitiesTask": {
+      "properties": {
+        "parameters": {
+          "properties": {
+            "projectName": {
+              "type": "string"
+            },
+            "deploymentName": {
+              "type": "string"
+            },
```

Comment:
could you please redefine these two properties as standalone parameters similar to `model-version`? so that you can set their serialized value to be `project-name` and `deployment-name` respectively.

Reply:
Actually they should be just renamed to `project-name` and `deployment-name`, no need to define them as standalone parameters. For example, look at how `model-version` is defined in other tasks.

Reply:
@deyaaeldeen these names are coming from Language APIs design. You have a good point that we might have them aligning with model-version, but this was not raised in the board review and it's too late now to change this.
This will require a service update and the release have been underway for almost 10 days already.

I'll make sure we have a discussion around this for TA's upcoming Language API and also upcoming previews.

Reply:
@kayousef actually, the service deployment already respects the proposed change here and gives use `project-name` and `deployment-name` so the change I am requesting is needed for the swagger only to match the deployment responses.

Reply:
I am marking this thread back as unresolved for now but let me know if that does not work.

Reply:
Fixed